.github/actions/bpftrace/start/action.yaml

```diff
@@ -24,8 +24,10 @@ runs:
             # bpftrace v0.20.1 doesn't seem to play well with Linux 4.19
             # https://github.com/bpftrace/bpftrace/issues/3011
             # Let's buy us some time, and keep installing v0.19.1 for the moment.
-            echo 'deb http://deb.debian.org/debian trixie main' >> /etc/apt/sources.list
-            apt update && apt install -y bpftrace -t trixie
+
+            curl -L https://github.com/bpftrace/bpftrace/releases/download/v0.19.1/bpftrace -o bpftrace
+            install -m 755 bpftrace /usr/local/bin/bpftrace
+
```

Comment:
Can't this commit (` bpftrace: install from release instead that using apt `) be merged in the previous one?

Reply:
It's probably also worth adding a follow-up issue, to clean this up a bit more.

I remember discussing with @giorio94 whether we should just bake `bpftrace` into the lvh-image instead. And as https://github.com/bpftrace/bpftrace/issues/3011 is resolved, maybe we don't need to hard-code the binary version any longer.

Reply:
I agree about merging this one into the previous commit for the moment. Installation from the releases is also more reliable compared to using apt, as in that case the version may change unexpectedly.

https://github.com/bpftrace/bpftrace/issues/3011 doesn't seem to have been released yet, Once, that's available, we could start installing newer versions of bpftrace again. 

Including the binary into the lvh-images is definitely an option, although at the end of the day we need to always download the same amount of data, just from a different source. One advantage of including it would be possibly installing different versions for different kernel versions, but I'm not sure if that's really necessary. Having it here instead would give a bit more flexibility, and allow to control the upgrades via renovate and test that they actually work before merging for increased confidence.